webui/react/src/pages/TrialDetails/TrialInfoBox.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import {
+  CheckpointState, ExperimentDetails, ExperimentOld, RunState, TrialDetails, TrialItem,
+} from 'types';
+import { generateOldExperiments } from 'utils/task';
```

Comment:
**nit:** up to you if you think this PR is a good to do it or not, but a small refactor of converting `generateOldExperiments` to the single version of `generateOldExperiment` and making the original `generateOldExperiments` call the single version fo `generateOldExperiment` n-times. Could be applied to all the generate[plural item] functions.

Reply:
I added a singular version of this. here we might want to also remove `as ExperimentOld`  as well but I think we should leave the rest for a separate work.